invenio_accounts/proxies.py

```diff
@@ -22,3 +24,15 @@
 current_datastore = LocalProxy(
     lambda: current_app.extensions['security'].datastore)
 """Proxy to the current Flask-Security user datastore."""
+
+
+def get_db_change_history():
+    """Proxy funtion to db change history."""
```

Comment:
I would add a quick note here something along the lines:

> The "g" object is local to the Flask application context. An app context is pushed on every 
request or CLI command. This means that the users change history is automatically cleared with on tear down of the app context. This matches up with the database session is also tied to the app context.

```